src/main/java/ca/bc/gov/chefs/etl/forms/pcdbi/decisionLog/model/PrimaryCareInitiatives.java

```diff
@@ -0,0 +1,73 @@
+package ca.bc.gov.chefs.etl.forms.pcdbi.decisionLog.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import ca.bc.gov.chefs.etl.constant.Constants;
+import ca.bc.gov.chefs.etl.core.model.IModel;
+
+public class PrimaryCareInitiatives implements IModel{
+
```

Comment:
I think this needs to have submissionId to point to the submission.

Reply:
PrimaryCareInitiatives  table was dropped. Using submissionId instead of confirmationId for every other tables